### PR TITLE
PBTA Histologies: Fusion summary base (4 of N)

### PR DESCRIPTION
**Pull request by kgaonkar6**
#### What scientific question is your analysis addressing?
| Module |  Reason |Brief Description | output
|--------|----------|--------|------
| [`fusion-summary`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion-summary)| adding 8 samples #749 and used in subtyping  | Generate summary tables from fusion files updated in  #812 and #816 and #821 and [comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/750#issuecomment-691529675) | `results/fusion_summary_embryonal_foi.tsv` (included in data download) <br> `results/fusion_summary_ependymoma_foi.tsv` (included in data download) <br> `results/fusion_summary_ewings_foi.tsv` <br> `results/fusion_summary_lgat_foi.tsv`

#### What GitHub issue does your pull request address?
#861 

#### Which areas should receive a particularly close look?
This is only a re-run for v18 RNAseq samples other modules that need to be rerun for subtyping are in #861 

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes

#### What types of results are included (e.g., table, figure)?
tables

#### What is your summary of the results?
8 new samples were added to output files from this module 
fusion_summary_embryonal_foi.tsv
fusion_summary_ependymoma_foi.tsv
fusion_summary_ewings_foi.tsv
fusion_summary_lgat_foi.tsv

#### Reproducibility Checklist

- [ ] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

- [ ] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [ ] The analytical code is documented and contains comments.

### CODE REVIEW

**Review by jharenza — APPROVED** [review]
Looks good - 8 samples added to the summary files.

**Review by jharenza — COMMENTED** [review]
Looks like this PR still needs a README update.

**Comment by kgaonkar6** [comment]
> Looks like this PR still needs a README update.
fusion-summary doesn't read in pbta-histologies file so I haven't updated the README, do you want to have an update in the module specifying to re-run per release?

**Review by cansavvy — COMMENTED** [review]
My review comments so far here are pretty similar to part 3's review https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/865#pullrequestreview-554612163

But at this point I'm having trouble figuring out what is new to this PR as differentiated from part 3. We've accumulated enough changes that its a bit tricky to review and keep track.

Might be better to revisit this after v18 gets in and parts 1-3 are merged. Or if you can point out and summarize the specific file changes, that could help too.

**Comment by kgaonkar6** [comment]
Thanks for the reviews @cansavvy.  Since we had to update multiple modules to run for base subtyping I'm specifying the modules being re-run of updated in the "What scientific question is your analysis addressing?" let me know if it would be better to specify it clearly some other way. 

The only changes from this PR are: 
8 new samples were added to output files: 
fusion_summary_embryonal_foi.tsv
fusion_summary_ependymoma_foi.tsv
fusion_summary_ewings_foi.tsv
fusion_summary_lgat_foi.tsv

**Comment by jharenza** [comment]
> But at this point I'm having trouble figuring out what is new to this PR as differentiated from part 3. We've accumulated enough changes that its a bit tricky to review and keep track.
> 
> Might be better to revisit this after v18 gets in and parts 1-3 are merged. Or if you can point out and summarize the specific file changes, that could help too.

Thanks @cansavvy! These are tricky, _but_... these PRs (1-7) should be reviewed before V18 release is merged, because the final result of all of these is the new histologies file that will be released in V18. It is a tad bit circular, but the most straightforward way we could think to do these updates within OpenPBTA since we had been doing them on the backend pre-each release. I found that the easiest way to spot changes was to go into the module titled in each PR, then look for new file change dates and then review those. Sorry this is a bit confusing.

I think we can also take your suggestion and point out the changes, too!

**Review by cansavvy — APPROVED** [review]
Now that part 1-4 are in, I can see that this is mostly (except one line) results changes which is what we'd expect based on samples being added. 👍 

I just have one question about the non-results line add since I wasn't expecting it but this looks good to me.

File: analyses/fusion_filtering/run_fusion_merged.sh [hunk 1]
```diff
@@ -106,6 +106,7 @@ Rscript 03-Calc-zscore-annotate.R --standardFusionCalls "${scratch_path}/standar
                                   --normalExpressionMatrix $normal_expression_file \
                                   --outputfile "${scratch_path}/standardFusionStrandedExp_QC_expression"
 
+if [ "$RUN_FOR_SUBTYPING" == 0 ]; then
```
Comment (cansavvy) [inline analyses/fusion_filtering/run_fusion_merged.sh:109]: I wasn't expecting this add. Is it something that was meant to be added previously but didn't get in?

Comment (kgaonkar6) [reply analyses/fusion_filtering/run_fusion_merged.sh:109]: hmm this script is not part of fusion_summary module but would have been part of #865 which is already merged now. I'll revert these analyses/fusion_filtering/* files to master.

Comment (kgaonkar6) [reply analyses/fusion_filtering/run_fusion_merged.sh]: ok reverted those files

Comment (cansavvy) [reply analyses/fusion_filtering/run_fusion_merged.sh]: Just double checking: These changes are all intentional: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/866/commits/ca25c8dc172a5c1669e791d099146a5d7c23c496

Comment (cansavvy) [reply analyses/fusion_filtering/run_fusion_merged.sh]: Looking back at your description, I think yes. I think we are all set.

Comment (kgaonkar6) [reply analyses/fusion_filtering/run_fusion_merged.sh]: correct, thanks for double checking these are intentional to update to the module in master . 

I must have missed merging the new changes from fusion_filtering-base to this branch which left these older file in this branch which was staggered from fusion_filtering_base branch.

**Pull request merged**